rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -244,8 +251,8 @@ for example, `Progressive Calls`.
     ]
 ```
 
-*Example.* Caller-to-Dealer `CALL` with MQTT payload. Specifying `"ppt_serializer": "mqtt"` means that 
-the original MQTT message payload is passed as WAMP payload message as is, without any transcoding, as is.
+*Example.* Caller-to-Dealer `CALL` with MQTT payload. Specifying `"ppt_serializer": "native"` means that 
+the original MQTT message payload is passed as WAMP payload message as is, without any transcoding.
```

Comment:
exactly! thus, "native" means "whatever the app/tech outside wamp defines", such as in case of mqtt is "whatever the app defines", since mqtt is leaving that fully to the app